inode/inode.go

```diff
@@ -373,55 +373,52 @@ func (vS *volumeStruct) inodeCacheDiscard() {
 		inodesToDrop = (vS.inodeCacheLRUItems * globals.inodeSize) - vS.inodeCacheLRUMaxBytes
 		inodesToDrop = inodesToDrop / globals.inodeSize
 		inodesToDrop += inodesToDrop / 4
-	}
-	for ((vS.inodeCacheLRUItems * globals.inodeSize) > vS.inodeCacheLRUMaxBytes) &&
-		(inodesToDrop > 0) {
-		inodesToDrop--
+		for ((vS.inodeCacheLRUItems * globals.inodeSize) > vS.inodeCacheLRUMaxBytes) &&
```

Comment:
I now accept what @bschatz-swift & @charmster are trying to say here that "users" don't really care about # of inodes but rather space... but I guess I see no advantage in internally calculating size everytime... especially since it is an approximation anyway.

Couldn't we just "compute" vS.inodeCacheLRUMaxItems by taking vS.InodeCacheLRUMaxBytes and dividing by globals.inodeSize at volume creation time?